### DISCUSSION TITLE
lobe能否跳过等待当前助手回答，直接开始下一个助手的对话？

### DISCUSSION THREAD

**Discussion by Alwaysbesideyou** [lobehub/lobehub#1845] [General Question]
在撰写论文的过程中，经常依赖AI助手的帮助来润色文本。通常，我会提交较长的文本段落给Azure的GPT-4处理，但是等待回应的时间较长。在这个过程中，我有时会想到其他问题想要咨询AI助手，我必须点击【停止】放弃当前对话，或者等待当前对话完成后才能继续进行下一个对话。

我想知道是否有可能在不影响当前正在处理的对话的情况下，同时开始下一个对话？
![image](https://github.com/lobehub/lobe-chat/assets/62324232/67e35169-1792-4cc9-950c-56469b7bf922)

**Comment by arvinxx** [comment]
理论上应该可以，但是需要做一些改造
